COPYRIGHT

```diff
@@ -30,6 +30,7 @@ PHPPrintIPP            1.3           GPL-2+                      Yes
 Restler                3.0           LGPL-3+                     Yes             Library to develop REST Web services
 TCPDF                  6.2.12        LGPL-3+                     Yes             PDF generation
 TCPDI                  1.0.0         LGPL-3+ / Apache 2.0        Yes             FPDI replacement
+Swift Mailer           5.4.2-DEV     MIT license                 Yes             Comprehensive mailing tools for PHP
```

Comment:
I really hope this is a mistake and you've not added a development version…

Reply:
The file `htdocs/includes/swiftmaile/lib/classes/Swift.php` indicates version `6.0.2`.
@frederic34 :
If this is the correct version, the COPYRIGHT file should be updated.
Please also consider adding swiftmailer via composer #3351 to simplify dependencies maintenance : https://packagist.org/packages/swiftmailer/swiftmailer
Please also see #10203
Thanks !